src/main/kotlin/com/autonomousapps/model/Coordinates.kt

```diff
@@ -115,7 +115,8 @@ sealed class Coordinates(
 @JsonClass(generateAdapter = false)
 data class ProjectCoordinates(
   override val identifier: String,
-  override val gradleVariantIdentification: GradleVariantIdentification
+  override val gradleVariantIdentification: GradleVariantIdentification,
+  val buildName: String? = null // Name of the build in a composite for which the project coordinates are valid
```

Comment:
We track the **build name**, so that we can decide, for **add** advices, if a `project(:path)` dependency is possible – which is only if **source** and **target** are in the same build.